life/modernized/18-chapter-18.adoc

```diff
@@ -60,79 +66,80 @@ that will uphold our ancient doctrines, and that without equivocation.
 There is a disposition in many who profess our ancient doctrines, to equivocate;
 and wherever there is equivocation there will be a going off.
 I think the number will be small that will stand;
-but after that there will be a gathering.`'
+but after that there will be a gathering.`"
 
-"`To a Friend who called to see him, he said in substance, '`I am glad you have come;
+To a Friend who called to see him, he said in substance, "`I am glad you have come;
 and I hope you will be careful to do what you find it your duty to do.
 I have labored a long time in our poor Society, but my labor has nearly come to a close,
 and I can say as the apostle said:
 '`My speech and my preaching were not in the enticing words of man`'s wisdom,
-but in demonstration of the Spirit and of power.`' The Friend, being about to leave,
-inquiring what he should tell his friends, he replied, '`Tell them,
+but in demonstration of the Spirit and of power.`'`" The Friend, being about to leave,
+inquiring what he should tell his friends, he replied, "`Tell them,
 I love all that love the Truth, and walk in it;
-but I cannot have fellowship with those who forsake it.`'
+but I cannot have fellowship with those who forsake it.`"
 
-"`22nd.--'`I do not think that I see anything in my way.
+22nd.--"`I do not think that I see anything in my way.
 All seems well.
 What a favor to be an inhabitant of that city that needs not the light of the sun,
 nor the moon to shine in it, for the glory of the Lord does lighten it,
 and the Lamb is the light thereof!
 Oh! if I could quietly pass away to that blessed inheritance, how glad I should be!
-I hope there is nothing in the way! "`My soul thirsts for God, for the living God;
-when shall I come and appear before him!`" "`As the deer pants after the water-brooks,
-so pants my soul after you, O God.`"`'
+I hope there is nothing in the way! '`My soul thirsts for God, for the living God;
+when shall I come and appear before him!`' '`As the deer pants after the water-brooks,
+so pants my soul after you, O God.`'`"
 
-"`About noon today the accumulation of phlegm in the
+About noon today the accumulation of phlegm in the
 throat seemed as though it must produce strangulation;
 and his friends were apprehensive that the period of his release had nearly come.
-In his struggles for life he said: '`I cannot stand it.
+In his struggles for life he said: "`I cannot stand it.
 I must go!
 Oh be honest!
 Oh be faithful!
-Joy forevermore appears great!`'
+Joy forevermore appears great!`"
 
-"`24th.--Today he remarked: "`Christ knows his own sheep, and his sheep hear his voice,
-and he leads them out, and goes before them; and a stranger they will not follow,
-but will flee from him, for they know not the voice of strangers.`"
+24th.--Today he remarked: "`'`Christ knows his own sheep, and his sheep hear his voice,
```

Comment:
I don't think this needs to be in single quotes (within the double quotes) because its not really a quotation from Scripture. It's more like a paraphrase, from Christopher's mouth. 👍🏻